### ISSUE TITLE
Author should validate against display name for the user

### ISSUE THREAD

**Issue by neilenns** [hlaueriksson/Community.PowerToys.Run.Plugin.Lint#3]
Right now the Author check validates against the github login, not the user's display name. This means in the PowerToys UI the author gets displayed as something like "neilenns" instead of the more appropriate "Neil Enns".

The display name for a user is available via the REST API at https://api.github.com/users/username (e.g. https://api.github.com/users/neilenns):

```json
{
  "login": "neilenns",
  "id": 9524118,
  "name": "Neil Enns",
}
```

Maybe this check can be updated to validate against either the login or the display name? The display name is more user friendly.

**Comment by hlaueriksson** [comment]
Yes, I agree!

**Comment by hlaueriksson** [comment]
This will be fixed in the next release

**Comment by neilenns** [comment]
I confirmed this passes now (using the current code in main built locally) against my DiscordTimestamp repo, which has the following in the plugin.json:

`"Author": "Neil Enns",`

Thanks!

**Comment by hlaueriksson** [comment]
This is now implemented in [0.3.0](https://www.nuget.org/packages/Community.PowerToys.Run.Plugin.Lint/0.3.0)
